src/app/public/modules/icon/icon-stack.component.html

```diff
@@ -0,0 +1,15 @@
+<span
+  class="sky-icon-stack fa-stack"
+  [ngClass]="size ? 'fa-' + size : ''"
+>
+  <i *ngIf="baseIcon"
+    aria-hidden="true"
```

Comment:
You can move the `aria-hidden` property to the parent `span` tag, and all children will inherit it.
> An element is considered hidden if it, or any of its ancestors are not rendered or have their aria-hidden attribute value set to true.
https://www.w3.org/TR/wai-aria-1.1/#aria-hidden